specification/http/src/main/scripts/org/kaazing/specification/http/rfc7235/auth.tests/errored.class/request.rpt

```diff
@@ -0,0 +1,11 @@
+#YWRtaW46YWRtaW4= : admin:admin   dGVzdDp0ZXN0 : test:test   am9lOndlbGNvbWU= : joe:welcome (base64)

```

Comment:
Please document the reason for the comments, and make them tidy.  I would suggest moving the comments closer to the point of usage.  Please remove base64 encoding comments for items not used in individual scripts.